torch/csrc/jit/mobile/backport_manager.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include <functional>
+#include <memory>
+#include <unordered_map>
+
+namespace c10 {
+struct IValue;
+}
+
+namespace caffe2 {
+namespace serialize {
+class IStreamAdapter;
+class ReadAdapterInterface;
+class PyTorchStreamWriter;
+class PyTorchStreamReader;
+} // namespace serialize
+} // namespace caffe2
+
+namespace torch {
+namespace jit {
+
+constexpr const char* kArchiveNameConstants = "constants";
+constexpr const char* kArchiveNameBytecode = "bytecode";
+constexpr const char* kArchiveNameVersion = "version";
+constexpr int64_t kBytecodeVersionV4 = 0x4L;
+constexpr int64_t kBytecodeVersionV5 = 0x5L;
```

Comment:
These should live in their implementation since I don't think any other code will use them.

Reply:
Ah good point. Move it to the implementation. It's not in their namespace though, because I think some of them will be shared. For example, `kBytecodeVersionV5` can be used by both `backport_v6_to_v5` and `backport_v5_to_v4`